services/ui_backend_service/api/utils.py

```diff
@@ -316,6 +318,11 @@ async def find_records(request: web.BaseRequest, async_table=None, initial_condi
     return web_response(status, res)
 
 
+def query_param_enabled(request: web.BaseRequest, name: str) -> bool:
+    """Parse boolean query parameter and return enabled status"""
+    return request.query.get(name, False) in ['True', 'true', '1', "t"]
+
+
```

Comment:
excellent cleanup. A lot of routes use this helper now so covering the intended behavior with an unit test might be benefitial, just to check that it returns a correct boolean even with unexpected parameters